projects/netgrif-components/src/lib/panel/task-panel/task-panel.component.html

```diff
@@ -51,7 +51,8 @@
         </div>
     </ng-template>
     <ng-template #taskPanelContent>
-        <div class="panel-main-content" fxLayout="column">
+        <div class="panel-main-content" fxLayout="column"
+             [ngStyle]="isForceOpen() && {'min-height': getContentMinHeight()}">
```

Comment:
is this legal? What Angular version are we running?
This comment on SO says that it does not work on Angular 14 anymore https://stackoverflow.com/questions/37051496/combine-ngstyle-with-condition-if-else#comment128339162_46089536